roles/nfd_operator_check_resource_statuses/tasks/get_cluster_nfd_operator_resources.yml

```diff
@@ -0,0 +1,99 @@
+---
+- name: Determine the OpenShift cluster version being used
+  shell: |
+    set -o pipefail;
+    oc version -o json | jq --raw-output '.openshiftVersion' | cut -d"." -f1-2
+  register: oc_version
```

Comment:
OpenShift version is already available with `{{ openshift_release }}`
https://github.com/openshift-psap/ci-artifacts/blob/master/roles/check_deps/tasks/main.yml#L32